### PR TITLE
made the policy in html

### PR DESCRIPTION
**Pull request by denispop9**
# Description

I added coverted the policy to html to be able to appear on the site.

## What

Added html.

## Why

So that the policy can appear on the site.

## How

By adding html.

## Verification Steps

What steps were taken to verify the change is working correctly

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by denispop9** [comment]
@laurafitzgerald @evanhearne can you check my commit? please :)

File: pages/about/policies.html [hunk 1]
```diff
@@ -271,10 +271,11 @@ <h1 class="ui header">
               <div class="three column row">
               <div class="column">
                 <h1 class="ui header">
-                  Relationship and Sexual Education Policy
+                  Relationship and Sexual Education
                 </h1>
                 <p>
-                   Preview Content goes here
+                   <strong><em>Relationship and Sexual Education Policy</em></strong>Mount Sion is a Catholic boys school, founded by Blessed Edmund Rice in 1802. Its students are educated with Catholic values as laid down in the teaching of the Church. The school fosters the development of relationships based on the love of God and the love of every human being...
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: @denispop9 just missing a space between `Policy` and `Mount` here.

File: pages/about/policies.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
         $('#healthsafetycontent').load("policies/healthsafety.html");
         $("#homeworkcontent").load("policies/homework.html");
         $("#criticalincidentmanagementcontent").load("policies/criticalincidentmanagement.html");
-        $("#relationshipsexualeducationcontent").load("policies/relationshipsexualeducation.html");
+        $("#relationshipsexualeducationcontent").load("policies/relationshipsexualed.html");
```
Comment (laurafitzgerald) [inline pages/about/policies.html]: @denispop9 what's the reason behind changing the file name?

Comment (denispop9) [reply pages/about/policies.html]: the file name was named 'relationshipsexulaed.html' so it wouldn't pop up when I pressed 'read me'.

**Comment by denispop9** [comment]
@laurafitzgerald I changed everything you asked me to I'm 100% confident I am done. Feel free to have a look and merge:)

**Comment by laurafitzgerald** [comment]
@denispop9 yep, this change is good. Merging now.

**Pull request merged**